src/mono/netcore/CoreFX.issues.rsp

```diff
@@ -5,795 +5,6 @@
 -notrait category=failing
 -notrait category=IgnoreForCI
```

Comment:
you should be able to completely remove all of these `-notrait category` now.

Reply:
done